lib/ui/src/modules/ui/components/stories_panel/stories_tree/tree_decorators.js

```diff
@@ -114,7 +114,7 @@ function HeaderDecorator(props) {
     };
   }
 
-  const name = highlightNode(node, style);
+  const name = highlightNode(node, style, theme.highlightColor, 'black');
```

Comment:
Can you please make text color customizable here as well? `black` doesn't play well with our `highlightColor` =(
<img width="147" alt="screenshot 2019-01-11 at 20 25 19" src="https://user-images.githubusercontent.com/6651625/51055265-b62fff80-15df-11e9-9b8e-9aaf13de2eb0.png">